airbyte-integrations/connectors/source-typeform/acceptance-test-config.yml

```diff
@@ -3,25 +3,30 @@ acceptance_tests:
   spec:
     tests:
     - spec_path: "source_typeform/spec.json"
+      backward_compatibility_tests_config:
+        disable_for_version: "0.1.12"
   connection:
     tests:
     - config_path: "secrets/config.json"
       status: "succeed"
+    - config_path: "secrets/config_oauth.json"
+      status: "succeed"
     - config_path: "integration_tests/invalid_config.json"
       status: "failed"
   discovery:
     tests:
     - config_path: "secrets/config.json"
+      backward_compatibility_tests_config:
+        disable_for_version: "0.1.12"
   basic_read:
     tests:
     - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/configured_catalog.json"
       empty_streams:
       - name: webhooks
         bypass_reason: "no data"
       expect_records:
         path: "integration_tests/expected_records.jsonl"
-      fail_on_extra_columns: true
+      fail_on_extra_columns: false
```

Comment:
why do you change this value?

Reply:
missed to change this value during testing, got it back to `true`